library/alloc/src/macros.rs

```diff
@@ -35,6 +35,8 @@
 ///
 /// [`Vec`]: crate::vec::Vec
 #[cfg(not(test))]
+#[doc(alias = "alloc")]
+#[doc(alias = "malloc")]
```

Comment:
I'm not sure to see what `alloc` and `malloc` aliases bring here.

Reply:
People may wonder: "How do I allocate a new slice of memory on the heap" to which the canonical answer is: "create a new vector". `vec!` is a way to create new vectors, which may very well be what a person searching for "malloc" is looking for.

Reply:
Ok, that seems like that a good reason.